src/OgGroupAudienceHelper.php

```diff
@@ -24,6 +24,15 @@ class OgGroupAudienceHelper {
    */
   const DEFAULT_FIELD = 'og_group_ref';
 
+  /**
+   * The name of the field which reference user to groups.
+   */
+  const USER_REFERENCE_FIELD = 'og_membership_reference';
+
+  /**
+   * The name of the field which reference non-user entities to groups.
+   */
+  const NON_USER_REFERENCE_FIELD = 'og_standard_reference';
```

Comment:
I agree the names should be changed, but for different reasons :)

We do need to have the fields - as they are acting as the reference between the entity and the group.

However we have 2 different fields based on the host entity (i.e. user or non-user).

`USER_REFERENCE_FIELD` sounds as if it is referencing a user, but in fact it just indicates it is _attached_ to a user.

so maybe it would be more explicit to name them:

* `GROUP_REFERENCE_FIELD_ON_USER`
* `GROUP_REFERENCE_FIELD_ON_NON_USER`